pyspark_ai/prompt.py

```diff
@@ -239,6 +302,23 @@
     prefix=SPARK_SQL_PREFIX,
 )
 
+SQL_CHAIN_EXAMPLES_RAG = [
+    sql_question3 + f"\nAnswer:\n```{sql_answer3}```",
+    sql_question4 + f"\nAnswer:\n```{sql_answer4}```",
+]
+
+SQL_CHAIN_PROMPT_RAG = PromptTemplate.from_examples(
+    examples=SQL_CHAIN_EXAMPLES_RAG,
```

Comment:
So `examples` is the question-answer set that will be stored in vector store and then searched against?